sample/application/applicatin_sample.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "<guid>",
+  "name": "<localization-key>",
+  "type": "<type>", // grid, grid_structure, browser, python, clr
+  // Depending of the type, we have different settings
+"configuration": {
+
+  }
```

Comment:
**Type: clr:**

```json
{
    "namespace": "...",
    "class" : "..."
    "method": "..."
}
```

**Type: python:**

```json
{
    "pathh": "...",
    "class" : "..."
    "method": "..."
}
```

**Type: grid:**

```json
{
    "grid": "...",
    "connection" : "..."
    "loadOnOpen": true,
     "refreshOnSelect": false
}
```

**Type: browser:**

```json
{
    "url": "...",
     "tab": "...."
}
```

Reply:
@froeh010  take a look at the possible configurations above.

Reply:
Can I find the enum to the types anywhere or is it yet to be created?

Reply:
I would represent this as string